src/msw/listctrl.cpp

```diff
@@ -103,9 +103,20 @@ namespace
 inline bool
 wxGetListCtrlSubItemRect(HWND hwnd, int item, int subitem, int flags, RECT& rect)
 {
+    int ret;
     rect.top = subitem;
     rect.left = flags;
-    return ::SendMessage(hwnd, LVM_GETSUBITEMRECT, item, (LPARAM)&rect) != 0;
+    ret = ::SendMessage(hwnd, LVM_GETSUBITEMRECT, item, (LPARAM)&rect);
```

Comment:
```suggestion
    int ret = ::SendMessage(hwnd, LVM_GETSUBITEMRECT, item, (LPARAM)&rect);
```
Prefer initializing variables when declaring them.